### PR TITLE
chore(ci): centralised release-drafter

### PR DESCRIPTION
**Pull request by h1dden-da3m0n**
### Description

As promised in  jellyfin/jellyfin-meta-plugins#3 here a demo of how to use a centralised Release Drafter configuration.

### Changes

* allow GH workflows to run on changes in .github/**
* simplify the dependabot config for github-actions
* change the Release drafter config to 'extend' the centralised version

### Issues

* n/a

**NOTE** I am not sure if we want to keep the `commit-message` dependabot config for nuget packages as is, since it will always end up as `fix`-prefix how it stands right now

### CODE REVIEW

**Review by crobibero — APPROVED** [review]

**Comment by h1dden-da3m0n** [comment]
I should have mentioned that this should be blocked until jellyfin/jellyfin-meta-plugins#3 is merged otherwise the config would be invalid

**Review by oddstr13 — APPROVED** [review]

**Comment by h1dden-da3m0n** [comment]
:crossed_fingers: everything works as expected and thank you for the merge :tada:

**Comment by h1dden-da3m0n** [comment]
FIY, I will open similar PRs in the other Plugins soon :tm:

**Comment by oddstr13** [comment]
Don't worry, I only did it to the repos where I accidentally still had direct merge perms :fearful: (path of least resistance), and then the one's I care the most about right now.

Plenty of plugins left to do for you too! I'm done for now :smile:

**Pull request merged**
